### PR TITLE
Update layout of user guide

### PR DESCRIPTION
**Pull request by w2vgd**
Provide a table of contents and number the sections

### CODE REVIEW

File: docs/UserGuide.md [hunk 1]
```diff
@@ -3,40 +3,74 @@ layout: page
...
+1. Let's try out your first command by adding a new property! <br>Type `add property n/Bishan t/Hdb a/Blk 150 Bishan Street 11 #02-101 p/570150 d/30-6-2021` into the command box and press <kbd>Enter</kbd> to execute it. After that, scroll down to the bottom of the property list to see your newly added property.
+1. Want to view the properties in order of their asking price? Type the command `sort property o/desc k/price` and press <kbd>Enter</kbd>. The property list will now be ordered in descending order, with the property having the highest asking price at the top.
+1. Now let's try adding a new appointment. <br>Type `add appointment n/Meet Jacob for dinner r/At Orchard Central's food court d/19-4-2021 t/1930` and hit the <kbd>Enter</kbd> button. Scroll down to the bottom of the appointment list to see your newly added appointment.
+1. Forgot the next appointment date to meet Simon? Let's try to find out! Type `find appointment simon` and press <kbd>Enter</kbd>. There it is! Your appointment with Simon is on Sep 20, 2021.
```
Comment (dvdweien) [inline docs/UserGuide.md]: would "Forgot when you are supposed to meet Simon again? Let's try finding it out!" be better?

**Review by dvdweien — COMMENTED** [review]
Thanks man!

File: docs/UserGuide.md (hunk 1 above)
Comment (w2vgd) [reply docs/UserGuide.md]: ok!

**Review by dvdweien — APPROVED** [review]
LGTM

**Pull request merged**